pyproject.toml

```diff
@@ -24,15 +24,11 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.10,<4.0"
-importlib-metadata = {version = "^4.0", python = "<3.9"}
-ansys-fluent-core = "~=0.29.dev0"
-pyvista = { version = ">=0.44.0", extras = ["jupyter"]}
+ansys-fluent-core = "==0.30.dev1"
```

Comment:
Does this mean that Pyfluent visualization 0.18.0 will require PyFluent 0.30.dev1?

Reply:
No, it'll use a stable release, probably v0.30.0

Reply:
@prmukherj OK, thanks for confirming.